src/actions/assets/CopcViewer.js

```diff
@@ -5,7 +5,11 @@ import i18n from "../../i18n";
 export default class CopcViewer extends AssetActionPlugin {
 
   get show() {
-    return this.component.isBrowserProtocol && this.asset.type === 'application/vnd.laszip+copc';
+    return this.component.isBrowserProtocol && (
+      this.asset.type === 'application/vnd.laszip+copc'
+      || this.asset.href.toLowerCase().includes('ept.json')
```

Comment:
I think we should use urijs to get the path component and then do an endsWith check. That will be more accurate.

Reply:
Regarding this comment for Potree, Copc, Protomaps, there are cases where data items are stored on a server with credentials encoded as SAS keys. Maybe relying on the below browser methods would be more like what you have in mind? 

```js
url = new URL ('http://ABC.com/folderpath/ept.json?test=abc')
// URL { href: "http://abc.com/folderpath/ept.json?test=abc", origin: "http://abc.com/", protocol: "http:", username: "", password: "", host: "abc.com", hostname: "abc.com", port: "", pathname: "/folderpath/ept.json", search: "?test=abc" }
check = url.pathname.endsWith('ept.json')
```

Reply:
Yes, indeed! We just use urijs everywhere in the code, so to keep it consistent you can use:

```
import URI from 'urijs';
const href = 'http://ABC.com/folderpath/ept.json?test=abc';
check = URI(href).filename() == 'ept.json'
```